plugins/localhost/src/lib.rs

```diff
@@ -46,17 +46,24 @@ type OnRequest = Option<Box<dyn Fn(&Request, &mut Response) + Send + Sync>>;
 
 pub struct Builder {
     port: u16,
+    host: Option<String>,
     on_request: OnRequest,
 }
 
 impl Builder {
     pub fn new(port: u16) -> Self {
         Self {
             port,
+            host: None,
             on_request: None,
         }
     }
 
+    pub fn host<H: AsRef<str>>(mut self, host: H) -> Self {
+        self.host = Some(host.as_ref().to_string());
```

Comment:
```suggestion
    // Change the host the plugin binds to. Defaults to `localhost`.
    pub fn host<H: Into<String>>(mut self, host: H) -> Self {
        self.host = Some(host.into());
```

Reply:
the diff looks a bit weird but the main change is AsRef -> Into (since we're turning it into a string anyway). Just added a short doc sentence too.